### PR TITLE
Updated powerbi_user_additional policy

### PR DESCRIPTION
**Pull request by gwionap**
## A reference to the issue / Description of it

https://github.com/ministryofjustice/data-platform/issues/1916 in Analytical Platform backlog

## How does this PR fix the problem?

PowerBI user needs athena and glue access so amending the policy to include them without resorting to blanket permissions

## How has this been tested?

We will be testing this out with users. Aside from that, a self-review

## Deployment Plan / Instructions

Should not do. This is a standalone policy only applicable to AP accounts

## Checklist (check `x` in `[ ]` of list items)

- [x] I have performed a self-review of my own code
- [x] All checks have passed
- [ ] I have made corresponding changes to the documentation
- [ ] Plan and discussed how it should be deployed to PROD (If needed)

### CODE REVIEW

**Review by julialawrence — DISMISSED** [review]
`restricted-admin` lite. 🤣

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 1]
```diff
@@ -819,110 +819,127 @@ data "aws_iam_policy_document" "powerbi_user_additional" {
...
+      "arn:aws:s3:::mojap-derived-tables/dev/models/*",
+      "arn:aws:s3:::mojap-derived-tables/seeds/*",
+    ]
+    actions = [
+      "s3:GetObject",
+      "s3:GetObjectAcl",
+      "s3:GetObjectVersion",
     ]
-    sid = "readonly"
+    sid = "S3Readnly"
```
Comment (ASTRobinson) [inline terraform/environments/bootstrap/delegate-access/policies.tf:836]: S3ReadOnly ?

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 2]
```diff
@@ -819,110 +819,127 @@ data "aws_iam_policy_document" "powerbi_user_additional" {
...
-      "arn:aws:s3:::alpha-accredited-programmes-review-2021",
       "arn:aws:s3:::alpha-athena-query-dump",
-      "arn:aws:s3:::mojap-athena-query-dump"
+      "arn:aws:s3:::mojap-athena-query-dump",
+      "arn:aws:s3:::alpha-everyone/*",
+      "arn:aws:s3:::alpha-postcodes/database/postcodes/*",
+      "arn:aws:s3:::alpha-postcodes/database/names/*",
+      "arn:aws:s3:::mojap-derived-tables/prod/models/*",
+      "arn:aws:s3:::mojap-derived-tables/dev/models/*",
+      "arn:aws:s3:::mojap-derived-tables/seeds/*",
```
Comment (ASTRobinson) [inline terraform/environments/bootstrap/delegate-access/policies.tf]: is this comma required?

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 3]
```diff
@@ -819,110 +819,127 @@ data "aws_iam_policy_document" "powerbi_user_additional" {
...
+    ]
+    sid = "List"
+  }
+
+  statement {
+    effect    = "Allow"
+    resources = ["arn:aws:s3:::aws-athena-query-results-*"]
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
```
Comment (ASTRobinson) [inline terraform/environments/bootstrap/delegate-access/policies.tf]: is this comma required?

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 4]
```diff
@@ -819,110 +819,127 @@ data "aws_iam_policy_document" "powerbi_user_additional" {
...
-    sid       = "ssm"
+    effect = "Allow"
+    resources = [
+      "arn:aws:s3:::alpha-athena-query-dump/$${aws:userid}/*",
+      "arn:aws:s3:::mojap-athena-query-dump/$${aws:userid}/*",
+    ]
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteObject",
```
Comment (ASTRobinson) [inline terraform/environments/bootstrap/delegate-access/policies.tf]: is this comma required?

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 5]
```diff
@@ -819,110 +819,127 @@ data "aws_iam_policy_document" "powerbi_user_additional" {
...
+      "glue:GetDatabase",
+      "glue:GetDatabases",
+      "glue:GetTable",
+      "glue:GetTables",
+      "glue:GetPartition",
+      "glue:GetPartitions",
+      "glue:BatchGetPartition",
+      "glue:GetCatalogImportStatus",
+      "glue:GetUserDefinedFunction",
+      "glue:GetUserDefinedFunctions",
```
Comment (ASTRobinson) [inline terraform/environments/bootstrap/delegate-access/policies.tf]: is this comma required?

File: terraform/environments/bootstrap/delegate-access/policies.tf [hunk 6]
```diff
@@ -819,110 +819,127 @@ data "aws_iam_policy_document" "powerbi_user_additional" {
...
+    ]
+  }
+
+  statement {
+    sid       = "AllowGetPutObject"
+    effect    = "Allow"
+    resources = ["arn:aws:s3:::aws-athena-query-results-593291632749-eu-west-1/*"]
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject",
```
Comment (ASTRobinson) [inline terraform/environments/bootstrap/delegate-access/policies.tf]: is this comma required?

**Review by ASTRobinson — APPROVED** [review]

**Pull request merged**